std/container/rbtree.d

```diff
@@ -20,6 +20,7 @@ Authors: Steven Schveighoffer, $(WEB erdani.com, Andrei Alexandrescu)
 module std.container.rbtree;
 
 import std.functional : binaryFun;
+import std.format;
```

Comment:
Unfortunately the `void toString(scope void delegate(const(char)[]) sink, FormatSpec!char fmt) const` doesn't work if `std.format` is a local import.